src/frontend/static/frontend/src/components/cgds-panel/NewCGDSDatasetForm.tsx

```diff
@@ -14,6 +14,7 @@ type RemoveSurvivalTupleCallback = (datasetName: NameOfCGDSDataset, idxSurvivalT
  * Component's props
  */
 interface NewCGDSDatasetFormProps {
+    newCGDSStudy: DjangoCGDSStudy,
```

Comment:
Agregar la siguiente documentación `/** CDGSStudy to take some fields to generate the CGDSDataset Mongo collection's name. */`